src/commands/createFunction/FunctionV2WizardContext.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { FunctionTemplateV2 } from '../../templates/FunctionTemplateV2';
+import { ParsedJob } from "../../templates/script/parseScriptTemplatesV2";
+import { IFunctionWizardContext } from './IFunctionWizardContext';
+
+export interface FunctionWizardV2Context extends IFunctionWizardContext {
```

Comment:
What is your opinion on naming this `V2FunctionWizardContext`. It looks worse, but reads better IMO. I'm on the fence 😄 

Also whatever we choose I'd want to be consistent and use v2 as a prefix everywhere.

Reply:
I don't really like having the V2 prefixed because it makes it kind of harder to read for me.

Though actually, what about `FunctionV2WizardContext`? It's actually the Function that's V2, not the FunctionWizard. WizardContext should be one word anyway.

Reply:
Yeah, that's better.